src/test/java/org/ladocuploader/app/utils/Page.java

```diff
@@ -41,6 +43,9 @@ public String getHeader() {
   }
 
   public void goBack() {
+    await().atMost(20, TimeUnit.SECONDS).until(
```

Comment:
Lets make this like 5 seconds. If it's taking longer than that, we should look into it

Reply:
@sree-cfa by default await takes 10 seconds and it was failing. It is for one specific test and it is flaky.

Reply:
i extended it to an atMost and set it to 20 minutes so that the flaky test would be less flaky.